package.json

```diff
@@ -21,9 +21,9 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --watchAll=false",
+    "start": "craco start",
```

Comment:
This is for the post process with tailwind & sass ( recommended from the tailwind official documentation ). With create-react app there is no way to modify postcss